evennia/typeclasses/models.py

```diff
@@ -331,7 +332,7 @@ def init_evennia_properties(self):
         by fetching them once.
         """
         for propkey, prop in self.__class__.__dict__.items():
-            if hasattr(prop, "__set_name__"):
+            if isinstance(prop, (AttributeProperty, TagProperty)):
```

Comment:
Please use `inherits_from` here instead; we want to support child classes.

Reply:
It looks like `inherits_from` chokes on anything that doesn't have an mro() (like functions), which raises exceptions and causes a bunch of test failures.

I believe isinstance() does check the entire class heirarchy (see docs: https://docs.python.org/3.9/library/functions.html#isinstance) so I think it might be the right choice here. Subclasses like DBField get picked up correctly.

Reply:
I set this PR back to using isinstance for now since inherits_from is breaking tests. Let me know what you think.